src/storage/page_buffer.c

```diff
@@ -16042,9 +16063,10 @@ pgbuf_page_maintenance_daemon_init ()
 {
   assert (pgbuf_Page_maintenance_daemon == NULL);
 
-  auto looper_interval = std::chrono::milliseconds (100);
-  pgbuf_Page_maintenance_daemon = cubthread::get_manager ()->create_daemon (cubthread::looper (looper_interval),
-				  new pgbuf_page_maintenance_daemon_task ());
+  auto looper = cubthread::looper (std::chrono::milliseconds (100));
```

Comment:
Use auto type in exceptional cases, when type is hard to determine and irrelevant, like iterators. In this case it is easy to specify the type, and it helps.

I think you used auto following my lead in vacuum.c. But I was also wrong to use it :). Please replace auto types in vacuum too.